### PR TITLE
Fix API search for Play1

### PR DESCRIPTION
**Pull request by cies**
It's broken now, and that's a pity as the docs are quite complete!

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @cies,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

**Comment by cies** [comment]
Done!

File: app/views/documentation/v1.scala.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         @for(version <- context.version) {
             <form id="search" method="get" action="//www.google.com/search">
                 <input type="search" name="q" placeholder="Search @version docs" results="10">
-                <input type="hidden" name="sitesearch" value="@req.host@reverseRouter.home(context.alternateLang, version.name)">
+                <input type="hidden" name="sitesearch" value="site:playframework.com@reverseRouter.home(context.alternateLang, version.name)">
```
Comment (mkurz) [inline app/views/documentation/v1.scala.html:17]: @cies Please remove the `site:`, it gets added from google automatically. Actually having it breaks the google search.

Comment (cies) [reply app/views/documentation/v1.scala.html]: Done!

**Pull request merged**